src/detail-page/components/tabs/Tabs.tsx

```diff
@@ -31,60 +31,66 @@ Because it isn't determined how they will look like it's type is unknown.
 It should be change when it will be determined */
 interface TabProps {
   categories: Category[];
-  learn: unknown;
-  jobs: unknown;
-  resources: unknown;
+  profession: Profession;
 }
 
-export const Tabs: React.FC<TabProps> = ({
-  categories,
-  learn,
-  jobs,
-  resources,
-}) => {
-  const [chosenCategory, setChosenCategory] = useState(0);
-  const match = useRouteMatch();
+const tabList = {
+  learn: 'What to learn',
+  jobs: 'jobs',
+  resources: 'resources',
+};
 
-  const getTabContent = (categoryName: String) => {
-    switch (categoryName) {
-      case 'jobs':
-        return <JobsTab content={jobs} />;
-      case 'learn':
-        return <LearnTab content={learn} />;
-      case 'resources':
-        return <ResourcesTab content={resources} />;
-      default:
-        throw new Error('Wrong category name!');
-    }
+const getTabContent = (categoryName: String) => {
+  switch (categoryName) {
+    case 'jobs':
+      return <JobsTab content={tabList.jobs} />;
+    case 'learn':
+      return <LearnTab content={tabList.learn} />;
+    case 'resources':
+      return <ResourcesTab content={tabList.resources} />;
+    default:
+      throw new Error('Wrong category name!');
+  }
+};
+
+export const Tabs: React.FC<TabProps> = ({ categories }) => {
+  const [activeTab, setActiveTab] = useState(0);
+  const { path, url } = useRouteMatch();
+  const history = useHistory();
+  const handleTabClicked = (category: Category, tabIndex: number) => {
+    setActiveTab(tabIndex);
+    history.push(`${url}/${category.name}`);
+  };
+  const syncTabWithPath = (categoryIdFromPath: string) => {
+    const selectedCategoryIndex = categories.findIndex((category) => category.id === categoryIdFromPath);
+    const index = selectedCategoryIndex === -1 ? 0 : selectedCategoryIndex;
+    setActiveTab(index);
   };
 
   return (
-    <StyledCard>
-      <StyledTabs indicatorColor="primary" value={chosenCategory}>
-        {categories.map((category, index) => {
-          const urlPath = index === 0 ? match.url : `${match.url}/${category.name}`;
-          return (
-            <Link to={`${urlPath}`} key={category.name}>
-              <Tab
-                text={category.text}
-                chosenTab={chosenCategory}
-                index={index}
-                onClick={setChosenCategory}
-              />
-            </Link>
-          );
-        })}
-      </StyledTabs>
-      {categories.map((category, index) => {
```

Comment:
Also here it's enough if we have a single Route (that is we don't have to `map`) - after all, we only want to render one TabPanel at a time! :)